cubids/cubids.py

```diff
@@ -1510,6 +1687,12 @@ def _order_columns(df):
 
     This ensures that KeyGroup and ParamGroup are the first two columns,
     FilePath is the last, and the others are sorted alphabetically.
+
+    Notes
```

Comment:
I'm open to alternative strategies for how to define these values

Reply:
Since they're only used here, we could just define them in a constant in this function. WDYT?